sky/templates/spot-controller.yaml.j2

```diff
@@ -39,6 +39,8 @@ run: |
 
 envs:
   SKYPILOT_USAGE_USER_ID: {{logging_user_hash}}
+  # skip cloud identity check for spot controller to avoid the overhead.
+  SKYPILOT_SKIP_CLOUD_IDENTITY_CHECK: 1
```

Comment:
This is retrieved at the beginning of `backend_utils._check_user_identity_no_lock()`, so that the spot controller will skip the checking of the user identity